cmd/containerd-shim-runhcs-v1/exec_hcs.go

```diff
@@ -200,8 +204,30 @@ func (he *hcsExec) Start(ctx context.Context) (err error) {
 		}
 	}()
 	if he.id == he.tid {
+
+		// Disable flushing if HCS doesn't support it, and it's necessary.
+		var handle syscall.Handle
+		handle, err = flushing.PreStartFlushDisable(
+			he.isWCOW,
+			he.spec.Windows.IgnoreFlushesDuringBoot,
+			he.spec.Windows.HyperV != nil,
+			he.tid,
+			he.id,
+			filepath.Join(he.spec.Windows.LayerFolders[len(he.spec.Windows.LayerFolders)-1], "sandbox.vhdx"))
```

Comment:
This logic only works for !`isHyperV` as well. Might want to comment that this side has to change as well.